src/ucs/datastruct/ptr_array.c

```diff
@@ -172,37 +180,128 @@ unsigned ucs_ptr_array_insert(ucs_ptr_array_t *ptr_array, void *value,
     }
 
     /* Get the first item on the free list */
-    elem_index = ptr_array->freelist;
-    ucs_assert(elem_index != UCS_PTR_ARRAY_SENTINEL);
-    elem = &ptr_array->start[elem_index];
+    element_index = ptr_array->freelist;
+    ucs_assert(element_index != UCS_PTR_ARRAY_SENTINEL);
 
-    /* Remove from free list */
-    ptr_array->freelist = ucs_ptr_array_freelist_get_next(*elem);
+    elem = &ptr_array->start[element_index];
 
-    /* Populate */
-    *placeholder_p = ucs_ptr_array_placeholder_get(*elem);
-    *elem = (uintptr_t)value;
-    return elem_index;
+    /* Remove from free list and populate */
+    ptr_array->freelist =
```

Comment:
looks like no need to split the line, should fit 80 sym

Reply:
Fixed.